merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/model/TaskSpecType.java

```diff
@@ -27,4 +27,27 @@ public UnconstructableTaskSpecException(final String message) {
       super(message);
     }
   }
+
+  /**
+   * This method must behave as though implemented as:
+   * {@snippet :
+   * return this.getValidationFailures(this.instantiate(activity));
+   * }
+   */
+  default List<String> validateArguments(final Map<String, SerializedValue> activity)
+  throws InvalidArgumentsException {
+    return this.getValidationFailures(this.instantiate(activity));
```

Comment:
Minor nitpick, but of the methods introduced to these interfaces, this is the only one that calls its parameter `activity`. I'd recommend naming it "arguments" here even if only because `instantiate()` (the first method in the composition this wraps) also calls it `arguments`.